src/MiDispositivoMIDI_V3.cpp

```diff
@@ -26,6 +26,15 @@
 #include <Adafruit_NeoPixel.h>
 #include <MIDIUSB.h>
 #include <TimerOne.h>
+/*
+ * Color Palette
+ * Generate with retinaConverter.py (Retina 2.0+ Palette)
+ * ------------------------------------------------------
+ */
+
+const byte _R[128] = {0, 61, 125, 190, 255, 255, 61, 125, 190, 255, 255, 61, 125, 190, 255, 255, 61, 125, 190, 255, 255, 61, 125, 190, 255, 255, 45, 93, 142, 223, 190, 28, 61, 93, 190, 125, 12, 28, 45, 158, 61, 0, 0, 0, 125, 0, 0, 0, 0, 125, 0, 0, 0, 0, 125, 0, 0, 0, 0, 125, 0, 0, 0, 0, 125, 0, 0, 0, 0, 125, 0, 0, 0, 0, 125, 0, 0, 0, 0, 125, 0, 0, 0, 0, 125, 0, 12, 28, 45, 158, 61, 28, 61, 93, 190, 125, 45, 93, 142, 223, 190, 61, 125, 190, 255, 255, 61, 125, 190, 255, 255, 61, 125, 190, 255, 255, 61, 125, 190, 255, 255, 36, 73, 109, 146, 182, 219, 255};
```

Comment:
Can you explain what is this doing? Whatever it is, I don't think it should be part of the library, looks quite specific so it should be part of an example.

Reply:
> Can you explain what is this doing? Whatever it is, I don't think it should be part of the library, looks quite specific so it should be part of an example.

I have made the ledstrip midi like launchpad on arduino pro micro and i want to add the exampke of my project if is possible:
[https://github.com/4dvn/Leonardo-Underlights/blob/master/Leonardo/WS2812B/WS2812B.ino](url)